src/core/qgsauxiliarystorage.h

```diff
@@ -197,6 +199,30 @@ class CORE_EXPORT QgsAuxiliaryLayer : public QgsVectorLayer
      */
     virtual bool deleteAttribute( int attr ) override;
 
+    /**
+     * Create if necessary a new auxiliary field for a PAL property and
+     * activate this property in settings.
+     *
+     * \param p The property to create
+     * \param providerId The id of the provider to use
+     * \param vlayer The vector layer
+     *
+     * \returns The index of the auxiliary field or -1
+     */
+    static int createProperty( QgsPalLayerSettings::Property p, const QString &providerId, QgsVectorLayer *vlayer );
```

Comment:
Please prefer longer, descriptive names

```py
QgsAuxiliaryLayer.createProperty(property=QgsPalLayerSettings.Bold)
```

reads much better than


```py
QgsAuxiliaryLayer.createProperty(p=QgsPalLayerSettings.Bold)
```

Reply:
Oops sorry, it's fixed